kubernetes-examples/src/main/java/io/fabric8/openshift/examples/DeploymentConfigExamples.java

```diff
@@ -94,7 +93,7 @@ public static void main(String[] args) throws InterruptedException {
 
       log("Done.");
     }finally {
```

Comment:
@manusa I think finally block can be removed from here as we are closing client in try block only, but I am not sure if doing that is correct or not, so asking here.

Reply:
There is no need to explicitly close client in finally block since try with resources handles it.

Reply:
So, do I remove the finally block or just the line that closes the client?

Reply:
As I already said in my previous comment, you need to remove line which closes the client. You should try to read about how try with resources work in Java

Reply:
No, actually I got your point about removing the line which closes the client, I was asking that is there any need for an empty finally block? I think we can remove that too.

Reply:
Yes, you're right. You can go ahead and remove finally block if it's empty.